polkadot/xcm/xcm-runtime-api/src/conversions.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Contains runtime APIs for useful conversions, such as between XCM `Location` and `AccountId`.
+
+use codec::{Decode, Encode};
+use scale_info::TypeInfo;
+use xcm::VersionedLocation;
+use xcm_executor::traits::ConvertLocation;
+
+sp_api::decl_runtime_apis! {
+	/// API for useful conversions between XCM `Location` and `AccountId`.
+	pub trait LocationToAccountApi {
+		/// Converts `Location` to `AccountId`.
+		fn convert_location(location: VersionedLocation) -> Result<sp_std::vec::Vec<u8>, Error>;
```

Comment:
```suggestion
	pub trait LocationToAccountApi<AccountId> {
		/// Converts `Location` to `AccountId`.
		fn convert_location(location: VersionedLocation) -> Result<AccountId, Error>;
```

Don't hate me :see_no_evil:  Sorry, I think I said `Vec<u8>` before? But by having a propoer type here, downstream will be able to decode the type easily using the metadata.

Reply:
:D :D fixed here: https://github.com/paritytech/polkadot-sdk/pull/4857/commits/e39e6e742e95c157e38515ab30a24f25f451c034